solarforecastarbiter/io/reference_observations/common.py

```diff
@@ -410,6 +410,20 @@ def site_name_no_network(site):
         return site.name
 
 
+def _make_fx_name(site_name, template_name, variable):
+    fx_name = f'{site_name} {template_name} {variable}'
+    # Some site names are too long and exceed the API's limits,
+    # in those cases. Use the abbreviated version.
+    if len(fx_name) > 63:
+        for old, new in (('Persistence', 'Pers.'), ('persistence', 'pers.')):
+            template_name = template_name.replace(old, new)
+        suffix = f'{template_name} {variable}'
```

Comment:
can the suffix ever be greater than 63 characters? If so, the while loop won't exit.